### ISSUE TITLE
Running list of topics for dicusssion/inclusion in dev guide: 

### ISSUE THREAD

**Issue by kellykapsar** [IBEEM-MSU/ibeem-devguide#1]
- Practical examples of commits (both scope of content committed and commit messages) in GitHub, including ecological examples 
- Establishing a set of workflow norms within a team (e.g., file structure(s) and location(s) and file naming conventions)
- Branching and when to do so
- R vs. Python (and/or strategies for teamwork with multiple coding languages) 
- Practical use of chatGPT in data anlaysis (with example prompts and discussions of when not to use chatGPT)

**Comment by kellykapsar** [comment]
- Setting up shared data folders (and having them sync with your local computer?)

**Comment by billspat** [comment]
I know I suggested using issues for this, but I think issues are best for short, targeted tasks that can be completed with code, and topics are broad, running and could use discussion.  I've added a new [ROADMAP](https://github.com/IBEEM-MSU/ibeem-devguide/blob/main/ROADMAP.md) file for this, which can be edited directly on github (but we just need to always pull before pushing!).   Focused issues can be generated from that roadmap doc. 

if all the ideas from this issue are copied into the roadmap doc, we can close this one, and start opening new issue. First issue: write out the roadmap.
